<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="jinto" failures="3" tests="4" name="Chrome Test" time="14.648" errors="0" timestamp="2025-03-28T09:03:19 IST">
  <testcase classname="com.hexaclothes.HexaClothesAutomation.Login" name="invalidEmailAndInvalidPassword" time="0.607">
    <failure type="java.lang.AssertionError" message="Test Failed:Login should have failed,but homepage loaded. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Test Failed:Login should have failed,but homepage loaded. expected [false] but found [true]
at com.hexaclothes.HexaClothesAutomation.Login.invalidEmailAndInvalidPassword(Login.java:72)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- invalidEmailAndInvalidPassword -->
  <testcase classname="com.hexaclothes.HexaClothesAutomation.Login" name="invalidEmailAndValidPassword" time="0.577">
    <failure type="java.lang.AssertionError" message="Test Failed:Login should have failed,but hompage loaded. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Test Failed:Login should have failed,but hompage loaded. expected [false] but found [true]
at com.hexaclothes.HexaClothesAutomation.Login.invalidEmailAndValidPassword(Login.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- invalidEmailAndValidPassword -->
  <testcase classname="com.hexaclothes.HexaClothesAutomation.Login" name="validEmailAndInvalidPassword" time="0.674">
    <failure type="java.lang.AssertionError" message="Test Failed:Login should have failed,but homepage loaded. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Test Failed:Login should have failed,but homepage loaded. expected [false] but found [true]
at com.hexaclothes.HexaClothesAutomation.Login.validEmailAndInvalidPassword(Login.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- validEmailAndInvalidPassword -->
  <testcase classname="com.hexaclothes.HexaClothesAutomation.Login" name="validUsernameAndValidPassword" time="0.589"/>
</testsuite> <!-- Chrome Test -->
